coalib/bearlib/abstractions/Linter.py

```diff
@@ -608,6 +613,9 @@ def linter(executable: str,
         created when ``generate_config`` is supplied. Useful if your executable
         expects getting a specific file-type with specific file-ending for the
         configuration file.
+    :param executable_check_fail_info:
+        Information that is provided together with the fail message from the
+        normal executable check. By default no additional info is printed.
```

Comment:
i suppose this one can be provided without the ``prerequisite_check_command``, right?